language-server/modules/langserver-stdlib/src/main/ballerina/functions_source1.bal

```diff
@@ -60,3 +60,8 @@ function function6(int param1, int param2) returns TestRecord3 {
     
     return rec;
 }
+
+public function function7(int param1, int param2) returns TestRecord4 {
```

Comment:
Due to this change some completion tests are failing. Shall we use an existing function. Otherwise there will be a lot of failures of completion tests which are using module1.

Reply:
Fixed